mdbenchmark/mdengines/namd.py

```diff
@@ -28,6 +28,9 @@
 def prepare_benchmark(name, relative_path, *args, **kwargs):
     benchmark = kwargs["benchmark"]
 
+    if not kwargs["multidir"] == 1:
+        raise NotImplementedError
```

Comment:
This is OK for now, but we should rather use our own `console.error` here. We don't want to throw a Python error at the user and instead just exit with an error message.

Reply:
```suggestion
        raise console.error("The NAMD-engine currently only supports '--multidir 1'")
```